spec/system/cohorts/student_report_spec.rb

```diff
@@ -485,4 +480,45 @@
       "This student's access to the course ended on #{time.strftime("%b %-d, %Y")}."
     )
   end
+
+  context "when user is a school admin" do
+    let!(:school_2) { create :school }
+    let!(:course_2) { create :course, school: school_2 }
+    let!(:cohort_2) { create :cohort, course: course_2 }
+    let!(:user_2) { create :user, school: school_2 }
+    let!(:school_2_student) { create :student, user: user_2, cohort: cohort_2 }
+
+    scenario "can access a student report" do
+      sign_in_user school_admin.user, referrer: student_report_path(student)
+      expect(page).to have_text(student.name)
+
+      expect(page).not_to have_text("Add a New Note")
+      expect(page).not_to have_button("Save Note")
+
+      # Can access student submissions tab
+      find("li", text: "Submissions").click
+
+      expect(page).to have_content(target_l1.title)
+    end
+
+    scenario "admin cannot access the student report belongs to school 2" do
```

Comment:
Refrain from mentioning implementation details of the test. Use descriptive language instead.

```suggestion
    scenario "admin cannot access the report of a student belonging to another school" do
```